<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresaveDialog</class>
 <widget class="QDialog" name="PresaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="saveInside">
       <property name="text">
        <string>Save pixels inside selection</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveOutside">
       <property name="text">
        <string>Save pixels outside selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="savePath">
       <property name="text">
        <string>Save segmentation path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="savePoints">
       <property name="text">
        <string>Save chosen points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveTextFile">
       <property name="text">
        <string>Save additional text file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PresaveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PresaveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
